GenerateRiverFlows/GenerateRiverFlows.py

```diff
@@ -0,0 +1,422 @@
+"""
+This python script is converted from Professor Simon Mathias(Professor of Environmental Engineering)'s
+script, which is used to generate river flows.
+                                                            Created by Jiada Tu
+                                                            13/01/2022
+"""
+
+
+# WARNING： The latest version of xlrd only supports .xls files.
+# Installing the older version 1.2.0 worked for me to open .xlsx files.
+
+import numpy as np
+import xlrd
+from datetime import date
+import math
+import os
+
+
+# here is converted from part of "GenerateRiverFlowsExample.m", which is used to read data from csv and xlsx files.
+# Specify start time
+t0 = date.toordinal(date(2010,1,1))+366
+
+def ModelFun(qp,Ep,dt,CatArea,X,F0):
```

Comment:
Please could you add docstrings for all of your functions so we know what each is doing?